measure/pom.xml

```diff
@@ -53,6 +53,8 @@ under the License.
         <scalaj.version>2.3.0</scalaj.version>
         <mongo.version>2.1.0</mongo.version>
         <curator.version>2.10.0</curator.version>
+        <postgresql.version>9.4.1212.jre7</postgresql.version>
```

Comment:
This version is compiled with java7 and is from 2016. Please use a more recent and stable version.

Reply:
> @lipzhu is there a valid JIRA ticket for this feature? If not then please create one, link it to this PR and also add a valid description to the PR.
> 
> Most of the PRs follow this process. Ref: #593

@chitralverma Thanks for you review, this PR try to resolve https://issues.apache.org/jira/browse/GRIFFIN-362.